crates/biome_configuration/src/linter/mod.rs

```diff
@@ -219,3 +220,65 @@ pub struct RuleWithOptions<T: Default> {
     pub level: RulePlainConfiguration,
     pub options: T,
 }
+
+#[derive(Clone, Copy, Debug, Eq, PartialEq, Hash)]
+pub struct RuleCode {
+    pub group: &'static str,
+    pub name: &'static str,
+}
+
+impl<'a> From<RuleCode> for RuleFilter<'a> {
+    fn from(RuleCode { group, name }: RuleCode) -> Self {
+        RuleFilter::Rule(group, name)
+    }
+}
```

Comment:
Have you considered making `name` a `Option<&'static str>`. This would allow us to create a `RuleFilter::Group` and run also rules that belong to a group:

```
biome lint --rule=complexity
```

Reply:
Yeah, I considered that. However, I gave up the idea because I am still unsure how we will evolve the way we refer to rules. We previously talked about removing the group from the rule name, and we also talked about rule domains or languages.

However, with a second thought, I accept that even if we change the way we refer to rules, it may still make sense to enable a particular group's rule.

Do you think we should rename the option? `--rule-filter`, `--filter-rule`, `--rules`?

Reply:
> Yeah, I considered that. However, I gave up the idea because I am still unsure how we will evolve the way we refer to rules. We previously talked about removing the group from the rule name, and we also talked about rule domains or languages.

I know. However, since things aren't final and we haven't taken any concrete steps towards this, we shouldn't hold ourselves from shipping new features.

> Do you think we should rename the option? `--rule-filter`, `--filter-rule`, `--rules`?

I fear that the word `filter` and the plural word `rules` might give an incorrect understanding and imply the possibility of accepting **multiple** rules/filters.  However, I think `--rule-filter` can work. 

Another suggestion: we could consider a `--group` option, and if both `--group` and `--rule` are passed, we can emit an "incompatible arguments" diagnostic.

Reply:
Or `--only`? Maybe it is too generic? Otherwise, I can extend the option and accept multiple rules?

Also, do you think that a group should enable all rules of the group, even the ones disabled in the configuration/disabled by default? I am more inclined to think we should respect the severity level of every rule when a group is passed?

Reply:
> Otherwise, I can extend the option and accept multiple rules?

I really depends on the scope of the feature and what you envisioned. I don't mind also accepting multiple rule filters. However, I wouldn't expose the term "filter" to the user because it's an internal terminology. Users know them as rules (and maybe groups).

> Also, do you think that a group should enable all rules of the group, even the ones disabled in the configuration/disabled by default? I am more inclined to think we should respect the severity level of every rule when a group is passed?

Yes, it makes sense to respect the configuration.